examples/FIL_streamz_pipeline.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<p>Provide some text that summarizes what this function does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>